Modules/@babylonjs/react-native/android/src/main/java/com/babylonreactnative/EngineView.java

```diff
@@ -125,7 +125,7 @@ public void takeSnapshot() {
             String encoded = "";
             if (copyResult == PixelCopy.SUCCESS) {
                 ByteArrayOutputStream byteArrayStream = new ByteArrayOutputStream();
-                bitmap.compress(Bitmap.CompressFormat.JPEG, 100, byteArrayStream);
+                bitmap.compress(Bitmap.CompressFormat.JPEG, 80, byteArrayStream);
```

Comment:
Just a thought, maybe it makes sense to expose this as an argument so that consumers can set whatever desired compression quality they like.

Reply:
We could do that, though I'm not sure its necessary with this change because there should not be any visual impact to the screenshots. This would also introduce a breaking change to anyone using screenshot, so I would recommend against this unless you feel strongly about it.

Reply:
I don't feel strongly either way.  I think eventually we want to replace this in our uses with the native capture apis in Babylon Native, so I think this is fine as is.